run.sh

```diff
@@ -9,22 +9,25 @@ TARGET=${TARGET:-armv7-unknown-linux-gnueabihf}
 TARGET_SSH=${TARGET_SSH:-pi@raspberrypi.local}
 PROFILE=${PROFILE:-debug}
 RUN=${RUN:-1}
+SUFFIX=${SUFFIX:-}
+BIN=${BIN:-publish-mqtt}
 
 if [ $PROFILE = release ]
 then
     time cross build --target $TARGET --release
 elif [ $PROFILE = debug ]
 then
-    time cross build --target $TARGET
+    time cross build --target $TARGET --bin $BIN
 else
     echo "Invalid profile '$PROFILE'"
     exit 1
 fi
 
-time rsync --progress target/$TARGET/$PROFILE/read-all-devices $TARGET_SSH:read-all-devices-next
-time rsync --progress target/$TARGET/$PROFILE/publish-mqtt $TARGET_SSH:publish-mqtt
+time rsync --progress target/$TARGET/$PROFILE/$BIN $TARGET_SSH:$BIN$SUFFIX
 
 if [ $RUN -eq 1 ]
 then
-    ssh $TARGET_SSH ./publish-mqtt
+    ssh $TARGET_SSH sudo systemctl stop $BIN.service || echo "Oh. Nevermind."
```

Comment:
Should this include the $SUFFIX?

Reply:
no. we only have a single publish-mqtt.service installed. If you're trying to run something other than publish-mqtt then this command will fail and that's fine.